### DISCUSSION TITLE
Prevent triggering of keyboard events if button is being held.

### DISCUSSION THREAD

**Discussion by gintarasgraciovas** [clientIO/joint#2718] [Q&A] [answered]
### Introduction

I have a problem with ui.Keyboard. Users have an ability to hold backspace/delete buttons which triggers a lot of events and as a result lots errors from the BE. Is it possible to somehow prevent such events?

### Steps to reproduce

1) add a few items to graph
2) select one of them
3) click and hold delete button
4) continue to hold button and select another item

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
Can you use the [repeat](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/repeat) `KeyboardEvent` property to detect this?

**Reply by gintarasgraciovas** [reply]
Works perfectly fine.
Cheers!
